src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -193,11 +193,15 @@ define(function (require, exports, module) {
                     BlobUtils.rename(oldPath, newPath);
                     BrambleEvents.triggerFileRenamed(oldPath, newPath);
                 }
-
-                callback();
+                FileSystemCache.refresh(function(){
```

Comment:
Add one blank line above this, so that it stands out from the `if` block above it.